comfy_cli/command/install.py

```diff
@@ -214,7 +214,9 @@ def execute(
     if skip_manager:
         print("Skipping installation of ComfyUI-Manager. (by --skip-manager)")
     else:
-        manager_repo_dir = os.path.join(repo_dir, "custom_nodes", "ComfyUI-Manager")
+        manager_repo_dir = os.path.join(
+            repo_dir, "custom_nodes", "comfyui-manager@nightly"
```

Comment:
We may need to change this. Depending on the version of comfyUI manager, the repo directory would change.